index.bs

```diff
@@ -823,16 +900,36 @@ The following example shows an [[#item-resource]] in the [[#ttl-representation]]
 
 ### Signed tree head ### {#signed-tree-head}
 
+The signed tree head for a register is available from the [[#register-proof-resource]], specifiying the proof-identifier as merkle:sha-256.
+
+The signed tree head for a register is the signed Merkle tree hash ([[RFC6962]] section 2.1) of a Merkle tree containing all entries in the register. 
```

Comment:
Not sure about "Merkle tree hash" as a term here, because it might refer to an intermediate node rather than the root. Maybe "Merkle root hash" or similar?